index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css" type="text/css" />
+    <title>Real food made Real Good!</title>
+</head>
+<!-- Header Section -->
+<header> Real food made Real Good!</header>
+<!-- Main section -->
+<main>
+    <div><img src="recipes/images/bon.png" alt="images"></div>
+    <h1> Real food made Real Good!</h1>
+    <!--Bottom Navigation section -->
+        <ul>
+            <li><a href="recipes/spinach_salad.html"> Spinach and Strawberry Salad </a></li>
+            <li><a href="recipes/berry.html"> Gordon's Berry Breakfast Drink</a></li>
+            <li><a href="recipes/cream_cheese.html"> Cream Cheese Frosting II </a></li>
+        </ul>  
```

Comment:
Please add a nav tag element in your header section. This makes up for good semantic flow and is suitable for screen readers